### PR TITLE
Fix documentation typo

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens, here's a low-priority review when you get a chance

File: doc/sphinx/Pacemaker_Explained/fencing.rst [hunk 1]
```diff
@@ -190,319 +190,316 @@ for ``pacemaker-fenced``.
...
+     - Type
+     - Default
+     - Description
+   * - .. _primitive_stonith_timeout:
+
+       .. index::
+          single: stonith-timeout (primitive instance attribute)
+
+       stonith-timeout
+     - :ref:`timeout <timeout>`
```
Comment (clumens) [inline doc/sphinx/Pacemaker_Explained/fencing.rst:208]: It looks like this one is marked as a timeout, but all timeouts are marked as integers.  Is there a reason for that?

Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/fencing.rst:208]: That appears to be how we're parsing them

**Comment by kgaillot** [comment]
Added commit on top to update regression tests

File: doc/sphinx/Pacemaker_Explained/fencing.rst (hunk 1 above)
Comment (clumens) [reply doc/sphinx/Pacemaker_Explained/fencing.rst:208]: Previously, something like pcmk_reboot_timeout was marked as a "time" and its default was "60s".  Now it's an "integer" with a default of "60".

Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/fencing.rst:208]: Yeah, I was sure I must be missing something, but I could never find where it was treated as anything but an integer number of seconds. The only place I could find it being parsed was https://github.com/ClusterLabs/pacemaker/blob/main/daemons/fenced/fenced_commands.c#L236

**Comment by kgaillot** [comment]
rebased on current main

**Review by clumens — APPROVED** [review]

**Pull request merged**